### PR TITLE
chore: .package-lock update MGX-1147

### PR DESCRIPTION
**Pull request by dominikfedor**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3024,321 +3065,377 @@
...
+      "integrity": "sha512-ihb3B0T/wJm1eUuArYP4lCTSEoZsClHhuWyfo/kMX3m/odpqNcPfsz5O2A3NT7dXCAgWPGDQGPqygCpgeniKMw==",
+      "engines": {
+        "node": ">=12.0.0"
+      }
     },
-    "node_modules/@metamask/sdk": {
-      "version": "0.16.0",
-      "resolved": "https://registry.npmjs.org/@metamask/sdk/-/sdk-0.16.0.tgz",
-      "integrity": "sha512-mJEeC3swF0/AXnX0cCX3F7T7vDGV3PLZ+vQIwXZeEpxspGmJxfoQKaJXoj9e/3q7FjYHc32fZ4U5/2q7+xiUmw==",
+    "node_modules/@metamask/json-rpc-engine/node_modules/@metamask/utils": {
```
Comment (olgakonsta) [inline package-lock.json:3123]: Didn't we remove the metamask?

File: package-lock.json [hunk 2]
```diff
@@ -3024,321 +3065,377 @@
...
-      "version": "0.16.0",
-      "resolved": "https://registry.npmjs.org/@metamask/sdk-communication-layer/-/sdk-communication-layer-0.16.0.tgz",
-      "integrity": "sha512-yM5tY5fSZatEr9GfZsQgz85s/hDcEIe0NFopQOfhSfZEqANb2ckMtEqu/Q6rKEat6wfnVDzY34PJt+psMUwI8A==",
-      "dependencies": {
-        "bufferutil": "^4.0.8",
-        "date-fns": "^2.29.3",
-        "debug": "^4.3.4",
-        "utf-8-validate": "^6.0.3",
-        "uuid": "^8.3.2"
+    "node_modules/@metamask/json-rpc-engine/node_modules/semver": {
```
Comment (olgakonsta) [inline package-lock.json:3142]: Do we still need this?

File: package-lock.json [hunk 3]
```diff
@@ -3024,321 +3065,377 @@
...
-        "socket.io-client": "^4.5.1"
+      "engines": {
+        "node": ">=10"
       }
     },
-    "node_modules/@metamask/sdk-communication-layer/node_modules/uuid": {
-      "version": "8.3.2",
-      "resolved": "https://registry.npmjs.org/uuid/-/uuid-8.3.2.tgz",
-      "integrity": "sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg==",
+    "node_modules/@metamask/json-rpc-engine/node_modules/uuid": {
```
Comment (olgakonsta) [inline package-lock.json:3153]: Same as above?

File: package-lock.json [hunk 4]
```diff
@@ -3024,321 +3065,377 @@
...
+      ],
       "bin": {
         "uuid": "dist/bin/uuid"
       }
     },
-    "node_modules/@metamask/sdk-install-modal-web": {
-      "version": "0.16.0",
-      "resolved": "https://registry.npmjs.org/@metamask/sdk-install-modal-web/-/sdk-install-modal-web-0.16.0.tgz",
-      "integrity": "sha512-j0LR1lTjhJhIqkAwzfC05daDReVpwdg76uf7lWtneuYq2JLbRIejfXprtzBGkKQUeEHYcJXxkrX23gh2N6WVEg==",
+    "node_modules/@metamask/json-rpc-middleware-stream": {
```
Comment (olgakonsta) [inline package-lock.json:3165]: Same as above :)

File: package-lock.json [hunk 5]
```diff
@@ -3024,321 +3065,377 @@
...
       "dependencies": {
-        "qr-code-styling": "^1.6.0-rc.1"
-      },
-      "peerDependencies": {
-        "i18next": "22.5.1",
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
-        "react-i18next": "^13.2.2",
-        "react-native": "*"
+        "@metamask/json-rpc-engine": "^8.0.2",
```
Comment (olgakonsta) [inline package-lock.json:3170]: Same as above

File: package-lock.json [hunk 6]
```diff
@@ -3024,321 +3065,377 @@
...
+        "@metamask/safe-event-emitter": "^3.0.0",
```
Comment (olgakonsta) [inline package-lock.json:3171]: Same as above

**Comment by olgakonsta** [comment]
And @dominikfedor please add the ticket number to the PR title :)

**Comment by dominikfedor** [comment]
@olgakonsta I applied the changes regarding naming cleanup. And yeah metamask is still used under the hood of `wagmi` also according to the package-lock:
```
"node_modules/@wagmi/connectors": {
      "version": "5.0.26",
      "resolved": "https://registry.npmjs.org/@wagmi/connectors/-/connectors-5.0.26.tgz",
      "integrity": "sha512-aGc3oDQPQwVqJr7S/7IU7rF0bA61OYXGPLzj30Y3MSmmEWXtAEgKpqkhIwiEdYQAMnlR3ukbqROq8qmUm/iYQg==",
      "dependencies": {
        "@coinbase/wallet-sdk": "4.0.4",
        "@metamask/sdk": "0.26.5",
        "@safe-global/safe-apps-provider": "0.18.3",
        "@safe-global/safe-apps-sdk": "9.1.0",
        "@walletconnect/ethereum-provider": "2.13.0",
        "@walletconnect/modal": "2.6.2",
        "cbw-sdk": "npm:@coinbase/wallet-sdk@3.9.3"
      },
```

**Review by olgakonsta — APPROVED** [review]
LGTM

**Pull request merged**